app/views/application/_multi_answer_row.html.erb

```diff
@@ -0,0 +1,27 @@
+<div>
+  <dt class="question">
+    <%= translate_with_appeal_or_application "check_answers.#{multi_answer_row.question}.question" %>
+  </dt>
+
+  <dd class="answer">
+    <ul class="list list-bullet">
+      <% multi_answer_row.value.each do |value| %>
+        <% if value.is_a?(String) && value.present? %>
```

Comment:
shouldn't we move this to a helper?

Reply:
I've moved the type checking to a helper method